lms/static/coffee/src/instructor_dashboard/student_admin.coffee

```diff
@@ -171,6 +180,83 @@ class StudentAdmin
+      full_success_message = _.template(success_message, {student_id: unique_student_identifier})
+      error_message = gettext("Error starting a task to rescore entrance exam for student '<%= student_id %>'. Make sure that entrance exam has problems in it and student identifier is correct.")
+      full_error_message = _.template(error_message, {student_id: unique_student_identifier})
+
+      $.ajax
+        dataType: 'json'
+        url: @$btn_rescore_entrance_exam.data 'endpoint'
+        data: send_data
+        success: @clear_errors_then -> alert full_success_message
+        error: std_ajax_err => @$request_response_error_ee.text full_error_message
+
+   # delete student state for entrance exam
+    @$btn_delete_entrance_exam_state.click =>
+      unique_student_identifier = @$field_entrance_exam_student_select_grade.val()
+      if not unique_student_identifier
+        return @$request_response_error_ee.text gettext("Please enter a student email address or username.")
+      send_data =
+        unique_student_identifier: unique_student_identifier
+        delete_module: true
+      success_message = gettext("Entrance exam state is being deleted for student '<%= student_id %>'.")
+      error_message = gettext("Error deleting entrance exam state for student '<%= student_id %>'. Make sure student identifier is correct.")
+      full_success_message = _.template(success_message, {student_id: unique_student_identifier})
+      full_error_message = _.template(error_message, {student_id: unique_student_identifier})
+
+      $.ajax
+        dataType: 'json'
+        url: @$btn_delete_entrance_exam_state.data 'endpoint'
+        data: send_data
+        success: @clear_errors_then -> alert full_success_message
+        error: std_ajax_err => @$request_response_error_ee.text full_error_message
+
+    # list entrance exam task history for student
+    @$btn_entrance_exam_task_history.click =>
+      unique_student_identifier = @$field_entrance_exam_student_select_grade.val()
+      if not unique_student_identifier
+        return @$request_response_error_ee.text gettext("Please enter a student email address or username.")
+      send_data =
+        unique_student_identifier: unique_student_identifier
+      error_message = gettext("Error getting entrance exam task history for student '<%= student_id %>'. Make sure student identifier is correct.")
+      full_error_message = _.template(error_message, {student_id: unique_student_identifier})
+
+      $.ajax
+        dataType: 'json'
+        url: @$btn_entrance_exam_task_history.data 'endpoint'
+        data: send_data
+        success: @clear_errors_then (data) =>
+          create_task_list_table @$table_entrance_exam_task_history, data.tasks
+        error: std_ajax_err => @$request_response_error_ee.text full_error_message
```

Comment:
I think it would be better to return an error message from the server and display that here. The client doesn't have enough context to really tell the user what has gone wrong.